docs/maintainers/spotify-reader.rst

```diff
@@ -0,0 +1,14 @@
+Debugging Spotify Reader
+========================
+
+To debug spotify reader issues, begin with checking logs of the container. The ListenBrainz admin panel has
```

Comment:
should we link to the admin panel? As far as we know it's locked down to only admins, but I'm not sure if we should expose its URL here? I guess we also have the source code open for it too, so...

Reply:
Linking sounds fine, will do.